### DISCUSSION TITLE
It is normal that the iso file patched by tiny11 weighs only 782 mb

### DISCUSSION THREAD

**Discussion by allhypr** [ntdevlabs/tiny11builder#259] [Q&A]
It's based on W11 24h2 Release
![proof](https://github.com/user-attachments/assets/9f118ae8-2f09-4d86-8cd8-01d86c3341d5)
Original W11 ISO file
![proof2](https://github.com/user-attachments/assets/c3074d9e-6e7f-48e8-8e6f-e840f40574f2)

**Comment by allhypr** [comment]
Also found a problem when running the ISO on a VM, but there's nothing in the logs
![VMError](https://github.com/user-attachments/assets/f2032efc-bde3-48d1-97b0-161bc42d72ef)

**Comment by Miiraak** [comment]
Hi, 
No is not normal, a problem has occured when you have tried to build the tinyISO. The output file is corrupted.
I've made mine with a 'Win11_24H2_French_x64.iso' dowload from [Microsoft.com Win11 download](https://www.microsoft.com/en-US/software-download/windows11) to compare with your result.

#### Compare result between .iso with 'tiny11maker.ps1' :
![Capture d'écran 2024-11-20 132149 ](https://github.com/user-attachments/assets/22c9227c-745e-400d-ab33-2f3a822cca1b)

#### Result with 'tiny11coremaker.ps1':
![Capture d'écran 2024-11-20 152130](https://github.com/user-attachments/assets/6400ed24-ad2b-4cd1-9898-aa7c878ff319)

As you can see, the coremaker, that remove more stuff than the regular maker, loose ≈ 50% of its weight.
The probleme you facing down with your VM is induce by the corrupted file, shown by error 0x8007000D... 

The solution is to retry to build your tiny11. 🐍

**Comment by rmbg58** [comment]
I had the same problem. I had 24H2 700M something, and tried 25H2 about 900M something. then never can pass the driver installation.
it's 5G something after building. I was expecting much less, but tried 2 times to get the iso almost same 5.4G and install it smoothly. you need to have enough space on your disk (C: drive) and enough running memory while building the tiny11.iso.
